src/components/books/Books.module.scss

```diff
@@ -1 +1,11 @@
 @import '../../styles/partials/variables';
+
+.section-container {
+  width: 100%;
+  margin: 0 auto;
+  padding-top: 2.375rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $lightGray;
+}
```

Comment:
[ SUGGESTED CHANGE ]
I highly recommend you do these few additions to your styles. It's not a major fix to ask for changes but I trust you will work on it 👍 


- [ ] It would be nice if you could add this style below to the section-container class to give a bit of freedom from the form to the bottom of the screen. 
`padding-bottom: 50px;`

**This is from the original design but your own design doesn't have that free space I highlighted in green**
![image](https://user-images.githubusercontent.com/50941074/193429386-62ddaf4e-d216-4349-aeb0-495fccea2bbf.png)